plugin/evm/vm.go

```diff
@@ -1271,6 +1279,50 @@ func (vm *VM) initBlockBuilding() error {
 	return nil
 }
 
+func (vm *VM) SubscribeToEvents(ctx context.Context) commonEng.Message {
```

Comment:
It seems unusual to me that this method wouldn't return an error. We call it over the gRPC (which we will need to drop errors for) and I'd have expected to return `ctx.Err()` if we are returning because we were cancelled.

Reply:
I was wondering what should be done in the case of a gRPC connection being severed to the rpcchainvm. I was told that this is a fatal error and shouldn't happen. 

I decided that the best tradeoff is to simply block indefinitely instead of propagating the error further up the call hierarchy. 

Regarding the context - the context we pass inside this method can only be cancelled by the caller, and in such a case, we return 0.